db/utility.js

```diff
@@ -93,8 +93,90 @@ const createOrder = async (order)=>{
     }
 }
 
+const prepareOrderIdList = (orderResult)=>{
```

Comment:
just curious - is there any advantage in having, **prepareOrderIdList**, **prepareItem**, **prepareOrderItem**, etc as **const**? Instead of making them a **function**?